src/course-unit/__mocks__/courseSectionVertical.js

```diff
@@ -318,10 +318,12 @@ module.exports = {
     ],
     has_changes: true,
     actions: {
```

Comment:
Please leave everything as it was. The data in this block does not apply to our changes

Reply:
Removed, thanks